### DISCUSSION TITLE
Quick command to send emails

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [pimalaya/himalaya#509] [Ideas]
this is useful to quickly send an email in case you have the parameters as environment variables

```
himalaya message send --host=$HOST --port=$PORT --username=$USERNAME --password=$PASSWORD --from=$FROM --to=$TO --subject=$SUBJECT --body=$BODY --attachment=$ATTACHMENT
```

**Comment by soywod** [comment]
What you ask for is a sendmail-compatible interface for the `message send` command, right? It is technically doable but it may require a bit of work. It would be faster to just configure a sendmail-compatible client and use it both in Himalaya (with the sendmail backend) and outside of Himalaya (with command-line arguments).

**Comment by soywod** [comment]
I close because I feel that this feature is out of scope of Himalaya. I am of course open to discussion, feel free to share.
